src/cli/commands/deploy.ts

```diff
@@ -0,0 +1,196 @@
+require("dotenv").config();
+
+import chalk from "chalk";
+import { spawn } from "child_process";
+import fs from "fs";
+import ora from "ora";
+import path from "path";
+import { DEFAULT_CONFIG } from "../../config";
+import { findSWAConfigFile, logger, readWorkflowFile } from "../../core";
+import { cleanUp, getDeployClientPath } from "../../core/deploy-client";
+
+const packageInfo = require(path.join(__dirname, "..", "..", "..", "package.json"));
+
+export async function deploy(deployContext: string, options: SWACLIConfig) {
+  const { SWA_CLI_DEPLOYMENT_TOKEN, SWA_CLI_DEBUG } = process.env;
+  const isVerboseEnabled = SWA_CLI_DEBUG === "silly";
+
+  if (options.dryRun) {
+    logger.warn("***********************************************************************", "swa");
+    logger.warn("* WARNING: Running in dry run mode. This project will not be deployed *", "swa");
+    logger.warn("***********************************************************************", "swa");
+    logger.warn("", "swa");
+  }
+
+  const frontendFolder = path.resolve(process.cwd(), deployContext);
+  logger.log(`Deploying front-end files from folder:`, "swa");
+  logger.log(`  ${chalk.green(frontendFolder)}`, "swa");
+  logger.log(``, "swa");
+
+  const apiFolder = path.resolve(process.cwd(), DEFAULT_CONFIG.apiPrefix!);
+  if (!options.apiLocation) {
+    if (fs.existsSync(apiFolder)) {
+      logger.warn(
+        `An API folder was found at ".${
+          path.sep + DEFAULT_CONFIG.apiPrefix!
+        }" but the --api-location option was not provided. API will not be deployed!`,
+        "swa"
+      );
+    } else {
+      logger.warn(`No API found. Skipping...`, "swa");
+    }
+  } else {
+    logger.log(`Deploying API from folder:`, "swa");
+    logger.log(`  ${chalk.green(apiFolder)}`, "swa");
```

Comment:
Message here doesn't show the API folder that is being used, it'll always be the default one.

Maybe line 30 needs to be updated to have the `apiFolder` better reflecting the structure.

Reply:
Fixed in the `ga` branch. Thank you @aaronpowell